index.html

```diff
@@ -31,19 +42,20 @@
         </div>
         <article>
           <section id="img"></section>
-        <article class="template" data-category="about">
-          <div class="byline">
+        <article class="template template" data-category="about">
+          <!-- <div class="byline">
             By <address><a class="author-name" href="">Author Name</a></address>
             published <time pubdate datetime="2000-01-01">Publish Time</time>
-          </div>
+          </div> -->
         </article>
       </section>
     </main>
     <footer>
-      <h2>Contact Kate</h2>
-      <h2>Resume</h2>
+      <h3 id="footer">Contact Kate</h3>
+      <h3 id="footer">Resume</h3>
```

Comment:
@kmzurlocuva per the HTML spec, only 1 unique ID per element/per page.  Anytime you need to use an ID (which should be sparingly) make sure you are using different ones - otherwise, try using a class.